src/components/Homepage/MonthlyGoodExplained.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+	<div class="monthly-good-explained">
+		<div class="row">
+			<h1 class="column small-12 text-center">
+				How does Monthly Good work?
+			</h1>
+			<div class="steps-container">
+				<div class="column small-12 medium-6 large-3 step-wrapper">
+					<span class="number">
+						1
+					</span>
+					<kv-icon name="hp-mg-1-signup" class="mg-image" />
+					<h2>
+						Sign up
+					</h2>
+					<div class="description-text">
+						to automatically help a borrower every month
+					</div>
+				</div>
+				<div class="column small-12 medium-6 large-3 step-wrapper">
+					<span class="number">
+						2
+					</span>
+					<kv-icon name="hp-mg-2-choose" class="mg-image" />
+					<h2>
+						Choose
+					</h2>
+					<div class="description-text">
+						how much and who you want to lend to - Kiva's got it from here
+					</div>
+				</div>
+				<div class="column small-12 medium-6 large-3 step-wrapper">
+					<span class="number">
+						3
+					</span>
+					<kv-icon name="hp-mg-3-receive" class="mg-image" />
+					<h2>
+						Receive
+					</h2>
+					<div class="description-text">
+						a personalized report on your impact once a month
+					</div>
+				</div>
+				<div class="column small-12 medium-6 large-3 step-wrapper">
+					<span class="number">
+						4
+					</span>
+					<kv-icon name="hp-mg-4-multiply" class="mg-image" />
+					<h2>
+						Multiply
+					</h2>
+					<div class="description-text">
+						your impact as Kiva continues to re-lend your repayments
+					</div>
+				</div>
+			</div>
+			<div class="column large-12 text-center learn-more-wrapper">
+				<router-link
+					:to="`/monthlygood`"
+					class="learn-more-text"
+					v-kv-track-event="[
+						'homepage',
+						'click-monthly-good-Learn more',
+						'homepage-monthly-good-learn-more-link',
+						'true'
```

Comment:
This 4th param is unnecessary. The param just resolves to NaN in kvTrackEvent and isn't registered in snowplow/looker anyway. Also remove from the WhyKiva component

Reply:
Removed the 4th parameter in monthlyGoodExplained and WhyKiva